airbyte-integrations/connectors/source-tripletex-api/source_tripletex_api/source.py

```diff
@@ -126,6 +126,17 @@ def path(
     ) -> str:
         return "ledger/account"
 
+class BalanceSheet(TripletexApiStream):
+    primary_key = "id"
+
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        return None
```

Comment:
The balance endpoint requires `dateFrom` and `dateTo` parameters, `next_page_token ` shouldn't return None.
Try to use the `Postings` stream as a template.

Reply:
As both Postings and Balance use dateFrom and DateTo could the PostingStream be renamed and use for both or do they have to be unique @RudyiVT?

Reply:
Shouldn't be any conflicts, we need to test it